typescript/src/string/str_repeat.test.ts

```
@@ -0,0 +1,14 @@
+import { str_repeat } from "./str_repeat";
+const str = "Hello";
+
+describe("string repeat", () => {
+  it("have string double", () => {
+    expect(str_repeat(str, 2)).toEqual("HelloHello");
+  });
+  it("empty string", () => {
+    expect(str_repeat(str, 0)).toEqual("");
+  });
+  it("invalid second parameter", () => {
```

"should throw when invalid parameter is passed"